src/theme/colors.js

```diff
@@ -13,6 +13,7 @@ const colors = {
   lightGray: "#E0E0E0",
   aquamarine: "#00FFFF",
   turquoise: "#31CCAA",
+  ceruleanBlue: "#026cdf",
```

Comment:
Let's not bring in this color. We can style the title for the card on our end as a link or just plain text.